Dockerfile

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
+
+LABEL Maintainer="Olliver Schinagl <oliver@schinagl.nl>"
+
+WORKDIR /src
+
+COPY . /src/
+
+# hadolint ignore=DL3008  We want the latest stable versions
+RUN apk add --no-cache \
+        bsd-compat-headers \
+        bzip2-dev \
+        cmake \
+        curl-dev \
```

Comment:
we'll need to add `check` here, so the unit tests will work

Reply:
check! (pun intended) I recall mentioning that I could not get the tests to work, or did I make them work and this changed? Regardless, I'll fix it.

Reply:
Ah, you're right. `check-dev` on Alpine.  `check-devel` on redhat, `check` on ubuntu/debian and on freebsd. 

CMake CTest support wasn't in 0.103, so when you first tried it probably did nothing. It was only added in 0.104 development a couple months ago.